app/src/test/resources/application-test.yml

```diff
@@ -49,5 +49,7 @@ bip:
   applicationName: "${BIP_APPLICATION_ISS}"
   applicationId: "${BIP_APPLICATION_ID}"
   stationId: "${BIP_STATION_ID}"
-truststore: "bipcert.jks"
-truststore_password: "vropassword"
\ No newline at end of file
+truststore: "${BIP_TRUSTSTORE:}"
```

Comment:
Should these be under bip?

Reply:
Good catch! I'm still testing it, so I put it this way for now. When it is tested, I'll put it under BIP. I should also revisit all the BIP environment variable setting after we get credentials for higher environments. Some of them may be shared and do not need to have for each environment.